### DISCUSSION TITLE
Display player controls under video

### DISCUSSION THREAD

**Discussion by already-used-name** [owncast/owncast#3897] [Q&A]
Hello, I'm trying to customize my instance with the player controls always visible and below the video itself instead of on top of it like an overlay, but so far all I've managed is to make the controls always visible with css. This obstructs the bottom of the video.

Is there a way to move the control bar under the video, either with css or javascript?

**Comment by gabek** [comment]
I'm pretty sure you'd have to build your own player controls that interface with the player. I can't think of a way you'd be able to move the existing controls outside the player.
